kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -277,24 +277,22 @@ def _init_and_run_proof(claim: KClaim) -> tuple[bool, list[str] | None]:
 
                     if simplify_init:
                         _LOGGER.info(f'Simplifying initial and target node: {claim.label}')
-                        _new_init, _ = kcfg_explore.cterm_simplify(new_init)
-                        _new_target, _ = kcfg_explore.cterm_simplify(new_target)
-                        if is_bottom(_new_init):
+                        new_init, _ = kcfg_explore.cterm_simplify(new_init)
+                        new_target, _ = kcfg_explore.cterm_simplify(new_target)
+                        if new_init.bottom():
                             raise ValueError(
                                 'Simplifying initial node led to #Bottom, are you sure your LHS is defined?'
                             )
-                        if is_bottom(_new_target):
+                        if new_target.top():
```

Comment:
`bottom()`?

Reply:
Okay, I see what you mean now @PetarMax :D Thanks